shared/fs/destination-picker/index.js.flow

```diff
@@ -0,0 +1,15 @@
+// @flow
```

Comment:
does this really need a split mobile/desktop impl?

Reply:
Maybe not; I was gonna do it like this for now, and try and see if I can get away without a split when I get to the mobile (will get to it right after these are merged).